pkgs/by-name/at/atuin/package.nix

```diff
@@ -70,6 +70,7 @@ rustPlatform.buildRustPackage (finalAttrs: {
       SuperSandro2000
       sciencentistguy
       _0x4A6F
```

Comment:
I am not sure if so many maintainers are really that great. Since r-vdp did many of the last updates I think it is fully justified to add him.

@0x4A6F @sciencentistguy do you want to stay maintainers?

Reply:
Yes, still interested and invested. And welcome r-vdp.